.github/workflows/cli-deploy.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deploy branch to heroku
+
+on:
+  push:
+    branches: [heroku-deploy]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [13.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Run client install and test
+        run: |
+          cd WatchParty/client/
+          npm install
+          npm run test
+        env:
+          CI: true
+      - name: Deploy to Heroku
+        env:
+          HEROKU_API_TOKEN: ${{ secrets.HEROKU_API_TOKEN }}
+          HEROKU_APP_NAME: "watch-party-test-web-client"
+        if: github.ref == 'refs/heads/master' && job.status == 'success'
```

Comment:
Also, take this out. We don't need this yet (we will later), for now it'll just keep this from every running.

Reply:
removed if